subsys/bluetooth/host/shell/bt.c

```diff
@@ -1009,7 +1009,31 @@ void subrate_changed(struct bt_conn *conn,
 		bt_shell_print("Subrate change failed (HCI status 0x%02x)", params->status);
 	}
 }
-#endif
+
+#if defined(CONFIG_BT_SHORTER_CONNECTION_INTERVALS)
+void conn_rate_changed(struct bt_conn *conn, uint8_t status,
+		       const struct bt_conn_le_conn_rate_changed *params)
+{
+	if (status == BT_HCI_ERR_SUCCESS) {
+		bt_shell_print("Connection rate parameters changed: "
+			       "Connection Interval: 0x%04x (%d us) "
```

Comment:
```suggestion
			       "Connection Interval: %u us "
```

Does it make sense to print the value in hex, when it's now just a value in US? Previous unit (1.25ms) sort of made sense to have both as the actual value, and as converted to milliseconds.